### PR TITLE
Added Wikipedia Search Module

### PR DESCRIPTION
**Pull request by tarunxsh**
Feature #53  added
wikipedia_search

### CODE REVIEW

File: modules/search/wikipedia.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+			links += run.links
+
+		links = list(set(links))
+		filtered_links = []
+		if links == []:
+			self.output('Without result')
+		else:
+			self.alert('Links')
+			for link in links:
+				if 'en.wikipedia.org/wiki' in link:
```
Comment (saeeddhqan) [inline modules/search/wikipedia.py]: replace it with:
`
for link in filter(lambda x: 'wikipedia.org/wiki' in link, links):
...
`

**Review by saeeddhqan — COMMENTED** [review]
It is really good. you used all of the sources.
But it will be great if you work on Wikipedia API.
This [repo](https://github.com/goldsmith/Wikipedia) is very helpful to show you how it works

File: modules/search/wikipedia.py (hunk 1 above)
Comment (tarunxsh) [reply modules/search/wikipedia.py]: Ok I will work on API.
Thanks

Comment (saeeddhqan) [reply modules/search/wikipedia.py]: Make sure to create a util class for wiki API.
About user_agent also needs to use a constant user_agent. Based on Wikipedia, the uagent should be like this:
`OWASP Maryam: github.com/saeeddhqan/maryam`
Thank you!

**Comment by tarunxsh** [comment]
wikipedia-api

**Comment by saeeddhqan** [comment]
I'll work on it as soon as I could.
Thank you very much!

**Pull request merged**